### PR TITLE
On edit focus on the input field

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Ext.ux.touch.grid/feature/Paging.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ Ext.define('Ext.ux.touch.grid.feature.Paging', {
     },
 
     handleGridPaint : function(grid) {
-        if (!(grid instanceof Ext.ux.touch.grid.View)) {
+        if (!(grid instanceof (Ext.ux.touch.grid.View || WorkOrder.view.GridView))) {
```
Comment (mitchellsimoens) [inline Ext.ux.touch.grid/feature/Paging.js:219]: You shouldn't be overriding the source

Comment (rolandschuetz) [inline Ext.ux.touch.grid/feature/Paging.js:219]: Oh, sry. The second commit was accidentally pushed.
I just wanted to push the editor.field.focus();

---

I had to override this, we are using Sencha Architect and it doesn't allow import of custom components. So the only way to get it working was to create a new DataView class (WorkOrder.view.GridView) and than make an override with your code for this class.

Comment (mitchellsimoens) [inline Ext.ux.touch.grid/feature/Paging.js:219]: You can remove a commit: http://christoph.ruegg.name/blog/2010/5/5/git-howto-revert-a-commit-already-pushed-to-a-remote-reposit.html

**Comment by rolandschuetz** [comment]
Oh, a revert on my repro automatically fixed that one.
No it's fine ;)

**Comment by rolandschuetz** [comment]
Oh, a pull request doesn't show to a specific commit, but to all changes in the repository.
Now I see why the second change was in it as well.

I've added a few more useful things.

**Comment by rolandschuetz** [comment]
I will be on holiday next two weeks, so I have to commit some code now... which shouldn't be in the pull request anymore... :-/

**Pull request closed without merging**